verifier/src/channel.rs

```diff
@@ -121,6 +125,19 @@ where
             // constraint queries
             constraint_commitment,
             constraint_queries: Some(constraint_queries),
+            // num partitions used in commitment
+            partition_size_main: air
+                .options()
+                .get_partition_option()
+                .partition_size::<E::BaseField>(air.context().trace_info().main_trace_width()),
+            partition_size_aux: air
+                .options()
+                .get_partition_option()
+                .partition_size::<E>(air.context().trace_info().aux_segment_width()),
+            partition_size_constraint: air
+                .options()
+                .get_partition_option()
+                .partition_size::<E>(air.context().num_constraint_composition_columns()),
```

Comment:
nit: I would probably define a variable `let partition_options = air.options().get_partition_option();` and then use it in this block of code. Would improve readability a bit.

Reply:
agreed, changed now